plugins/jdg70/src/main/resources/server.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:4.0">
```

Comment:
From my point of view, using the default server configs for each plugin would be really useful. That's all I am doing at this point is copying in the default and avoiding this step when creating a new plugin would be helpful. What we do for changes to the default is a separate discussion. I'm not opposed to adding XML transformation but using a separate config file in the `etc` project is also a possibility.